samples/meetings-visual-indicator/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-visual-indicator-nodejs",
```

Comment:
Wrong name:

Update it to officedev-microsoft-teams-samples-**meetings**-visual-indicator-nodejs